sites/website/src/docs/design-systems/creating-a-design-system.md

```diff
@@ -0,0 +1,124 @@
+---
+id: creating-a-design-system
+title: Creating a Design System
+sidebar_label: Creating a Design System
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/creating-a-design-system.md
+---
+Creating a design system involves two key steps:
+1. Determine what information the Design System should hold
+2. Creating a [DesignSystemProvider](/docs/api/fast-foundation.designsystemprovider) to communicate the Design System
+
+The information the Design System should hold will be dictated by a project's design philosophies and capabilities and can vary widely from project to project. Therefore, this documentation will focus on creating and configuring a [DesignSystemProvider](/docs/api/fast-foundation.designsystemprovider) and not data in the Design System.
```

Comment:
"The Design System holds information dictated by a project's design guidelines, philosophies, and capabilities which may vary widely across projects."